### ISSUE TITLE
Request: provide github organisation export

### ISSUE THREAD

**Issue by mariomadproductions** [yuzu-emu/yuzu#13187]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

Request: provide github organisation export (with just the public data of course). `git clone --mirror` only includes the git repo, not issues, PRs etc.

### Why would this feature be useful?

In case of a takedown request.

**Comment by boehs** [comment]
Context: https://dockets.justia.com/docket/rhode-island/ridce/1:2024cv00082/56980

**Comment by Techwizz-somboo** [comment]
This is a great idea! I firmly believe this would be the best approach.

**Comment by mariomadproductions** [comment]
> Edit: I suggest just doing https://github.com/mattduck/gh2md on your own computer

Yeah there are some tools like that, unfortunately none of them seem that good and/or seem to rely on github's rate limited/TOS-limited API. I do think a tool like that is needed though, just for repo archiving in general - the owner won't always be around/willing to help.
